heft-plugins/heft-dev-cert-plugin/tsconfig.json

```
@@ -0,0 +1,8 @@
+{
+  "extends": "./node_modules/@rushstack/heft-node-rig/profiles/default/tsconfig-base.json",
+
+  "compilerOptions": {
+    "types": ["node"]
+  },
+  "include": ["src/**/*.ts"]
```

```suggestion
  }
```

This is already in [the base config](https://github.com/microsoft/rushstack/blob/master/rigs/heft-node-rig/profiles/default/tsconfig-base.json#L26).